src/register/components/config.tsx

```diff
@@ -20,12 +20,14 @@ export interface Config {
   mode: string;
   binary: string;
   source: string;
+  rtv: string;
```

Comment:
Should we then also have LTS here?

Reply:
If I understand correctly LTS maps to a specific RTV, so it can already load LTS by cross-referencing.

Maybe this should be more of a dropdown?

```
[ cdn.ampproject.org   v ]   [ stable            v ]   RTV ______________
                             |---------------------|
                             |   by RTV            |
                             |   beta              |
                             | ✔ stable            |
                             |   long term stable  |
                             +---------------------+
```

Granted, `stable` and `beta` switch cookies so they would only work when served from the CDN itself.

Reply:
Cool. Makes sense. I meant that we could download the versions file (like other caches can do) and use it here as a dropdown. But we can always do it later. Let's go simple for now.

Reply:
Got it. In that case we can probably merge into an auto-complete drop-down.

Leaving as-is with a single input for RTV for now.

FMI: is the versions file available publicly?

Reply:
Should be.